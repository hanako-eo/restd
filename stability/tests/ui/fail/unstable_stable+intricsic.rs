#![feature(staged_api)]
#![stable(feature = "root", since = "1.0.0")]
#![allow(internal_features)]

#[stability::unstable(
    feature = "test",
    issue = "none",
    reason = "sd",
    constness,
    stabilisable,
    intrinsic
)]
const fn test3() {}

fn main() {}
