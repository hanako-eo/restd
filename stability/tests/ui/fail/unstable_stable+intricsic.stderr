error: cannot have intrinsic and stabilisable at the same time
  --> tests/ui/fail/unstable_stable+intricsic.rs:5:1
   |
5  | / #[stability::unstable(
6  | |     feature = "test",
7  | |     issue = "none",
8  | |     reason = "sd",
...  |
11 | |     intrinsic
12 | | )]
   | |__^
   |
   = note: this error originates in the attribute macro `stability::unstable` (in Nightly builds, run with -Z macro-backtrace for more info)
